File: CotuTalk_Program/app/src/main/java/com/example/cotutalk_program/AcessoAPI/data/PostagemUI.kt
package com.example.cotutalk_program.AcessoAPI.data

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class PostagemUI (
    val IdPostagem : Int,
    val Conteudo: String,
    val Usuario: Usuario,
    val Grupo: Grupo,
    val qtsCurtidas: Int,
    val qtsRespostas: Int
) : Parcelable

File: api/TWTodos/Models/PostagemUI.cs

public class PostagemUI
{
    public int IdPostagem { get; set; }
    public required string Conteudo { get; set; }
    public required Usuario Usuario { get; set; }
    public required Grupo Grupo { get; set; }

    public int qtsCurtidas { get; set; }
    public int qtsRespostas { get; set; }
}